### PR TITLE
Remove multithreading

### PR DESCRIPTION
**Pull request by Robin5605**
Rust multithreading bad

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -27,9 +27,9 @@ fn scan_package(client: &DragonflyClient, job: Job) -> ScanResult {
                 PackageScanResults::new(job.name, job.version, results, job.hash);
             let body = package_scan_results.build_body();
 
-            Ok(body)
+            ScanResult::Ok(body)
```
Comment (jonathan-d-zhang) [inline src/main.rs:30]:
```suggestion
            Ok(body)
```

File: src/main.rs [hunk 2]
```diff
@@ -27,9 +27,9 @@ fn scan_package(client: &DragonflyClient, job: Job) -> ScanResult {
...
         }
-        Err(err) => Err(SubmitJobResultsError {
+        Err(err) => ScanResult::Err(SubmitJobResultsError {
```
Comment (jonathan-d-zhang) [inline src/main.rs:32]:
```suggestion
        Err(err) => Err(SubmitJobResultsError {
```

**Review by jonathan-d-zhang — APPROVED** [review]
lgtm

**Review by AbooMinister25 — APPROVED** [review]
I haven't run the program to check whether it works, but everything does look good.

**Pull request merged**